.github/workflows/ci.yml

```diff
@@ -103,6 +107,11 @@ jobs:
         run: |
           tee -a ~/.bash_profile <<<'export PATH="$pythonLocation/bin:$PATH"'
 
+      - if: ${{ matrix.python-version == '3.11' }}
+        # install numba pre-release for python 3.11
+        run: |
+          pip install numba==0.57.0rc1
```

Comment:
Is there a way to make this "0.57rc1 or later"?

Reply:
Yes, but I think I'd rather wait until the release is out and just remove this workaround then

Reply:
Numba 0.57 has been released (at least on PyPI, the conda-forge release is underway), so this was removed